packages/text-input/src/TextInput.tsx

```diff
@@ -218,20 +218,25 @@ const colorSets: Record<Mode, ColorSets> = {
   },
 } as const;
 
-function getStatefulInputStyles(state: State, optional: boolean, mode: Mode) {
+function getStatefulInputStyles(
+  state: State,
+  optional: boolean,
+  mode: Mode,
+  disabled: boolean,
+) {
```

Comment:
suggestion: In VSCode, right-click -> Refactor -> Convert parameters to destructured object

Reply:
How cool!!!

Reply:
Thanks for the tip